config/prod/kristen-bucket.yaml

```diff
@@ -0,0 +1,49 @@
+# Provision a general S3 bucket or a Synapse External Bucket
+# http://docs.synapse.org/articles/custom_storage_location.html
+template_path: "remote/s3-bucket-v2.yaml"
+stack_name: "kristen-bucket"
+sceptre_user_data:
+  # (Optional) Synapse IDs for owner.txt file
+  SynapseIDs: ["1968150"]  # Kristen Dang
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "CompOnc"
```

Comment:
@kkdang are these dept and project settings correct?

Reply:
Yes!